rfcs/text/0005/host.yml

```diff
@@ -0,0 +1,61 @@
+- name: host
+  fields:
+    - name: cpu.usage
+      type: scaled_float
+      scaling_factor: 1000
+      level: extended
+      short: Percent CPU used with scaling factor of 1000.
+      description: >
+        Percent CPU used with scaling factor of 1000. This value is normalized by
```

Comment:
Same here, instead of starting with the scaling factor, I would suggest mentioning that in the last sentence of the long description.